files/ru/learn/performance/index.html

```diff
@@ -1,5 +1,5 @@
 ---
-title: Web performance
+title: Производительность веб-сайтов
```

Comment:
Сделал более общий заголовок, так как веб покрывает не только сайты, но и приложения.

```suggestion
title: Веб-производительность
```